automatic/googlechromebeta/update.ps1

```diff
@@ -0,0 +1,37 @@
+﻿import-module au
+import-module "$PSScriptRoot\..\..\scripts\au_extensions.psm1"
+
+$releases = 'https://omahaproxy.appspot.com/all?os=win&channel=beta'
+$paddedUnderVersion = '57.0.2988'
+
+function global:au_BeforeUpdate {
+  $Latest.Checksum32 = Get-RemoteChecksum $Latest.URL32
+  $Latest.Checksum64 = Get-RemoteChecksum $Latest.URL64
+}
+
+function global:au_SearchReplace {
+  @{
+    ".\tools\chocolateyInstall.ps1" = @{
+      "(?i)(^\s*url\s*=\s*)('.*')" = "`$1'$($Latest.URL32)'"
+      "(?i)(^\s*url64bit\s*=\s*)('.*')" = "`$1'$($Latest.URL64)'"
+      "(?i)(^\s*checksum\s*=\s*)('.*')" = "`$1'$($Latest.Checksum32)'"
+      "(?i)(^\s*checksum64\s*=\s*)('.*')" = "`$1'$($Latest.Checksum64)'"
+      "(?i)(^[$]version\s*=\s*)('.*')" = "`$1'$($Latest.RemoteVersion)'"
+    }
+  }
+}
+
+function global:au_GetLatest {
+  $release_info = Invoke-WebRequest -Uri $releases -UseBasicParsing
+  $version = $release_info | % Content | ConvertFrom-Csv | % current_version
+
+  @{
+    URL32 = 'https://dl.google.com/tag/s/dl/chrome/install/beta/googlechromebetastandaloneenterprise.msi'
+    URL64 = 'https://dl.google.com/tag/s/dl/chrome/install/beta/googlechromebetastandaloneenterprise64.msi'
+    Version = Get-FixVersion $version -OnlyFixBelowVersion $paddedUnderVersion
+    RemoteVersion = $version
+    PackageName = 'GoogleChromeBeta'
```

Comment:
Please remove the package name, it is only necessary if there is previous versions have the casing is different (or isn't named the same as parent folder)